docs/marko-testing-library/api.md

```diff
@@ -0,0 +1,177 @@
+---
+id: api
+title: API
+---
+
+`Marko Testing Library` re-exports everything from `DOM Testing Library` as well
+as these methods:
+
+- [`render`](#render)
+- [`cleanup`](#cleanup)
+
+---
+
+## `render`
+
+```javascript
+function render(
+  template, // A Marko template to render
+  input, // Input for the above template
+  options // You won't often use this, expand below for docs on options
+):
+```
+
+Render into a container which is appended to `document.body`.
+
+```javascript
+import { render } from '@marko/testing-library'
+import MyTemplate from './my-template.marko'
+
+render(MyTemplate)
+```
+
+```javascript
+import { render, cleanup } from '@marko/testing-library'
+import 'jest-dom/extend-expect'
+import Greeting from './greeting.marko'
+
+afterEach(cleanup)
+
+test('renders a message', async () => {
+  const { container, getByText } = await render(Greeting, { name: 'Marko' })
+  expect(getByText(/Marko/)).toBeInTheDocument()
+  expect(container.firstChild).toMatchInlineSnapshot(`
+    <h1>Hello, Marko!</h1>
+  `)
+})
+```
+
+> Note
+>
+> The [cleanup](#cleanup) function should be called between tests to remove the
+> created DOM nodes and keep the tests isolated.
+
+## `render` Options
```

Comment:
Docs looks great and solid! 👍 

It's a bit weird, thou, that the "render options" section uses the same heading as the parent "render" heading.
I'd suggest setting "render Options" as a third-level heading, and "container" as a fourth-level one.

😃 


```suggestion
### `render` Options
```

Reply:
Excellent catches!
Updated the PR based on your comments.

Thanks for the review! 😄

Reply:
I realized I missed this part:

> Same thing with "render result" and its children.

I think having those as a higher heading is still useful as it causes it to show up in the side nav (and these are probably the most important parts of the lib). This is also what the `react-testing-library` docs have setup (https://github.com/testing-library/testing-library-docs/blob/79a0c1b8127d30b04182fbc528ebcb34adee3a0e/docs/react-testing-library/api.md#L116).

Reply:
Totally right, I was thinking about that while typing, actually 🤔 Let's follow RTL example, then! If we ever feel the urge to change it we can make sure we update each framework docs :)

Reply:
I still like moving `render Options` as a sub heading, since in this package there is only one option and honestly it would be very rare that people use it. Could promote it to a higher heading (as is done in the react docs) if more options become available, but I don't see that happening.

In my last commit I updated the PR to do as I said above, but I left the `render Result` heading alone.

I think I'm happy with it now 😄